cf-check/backup.c

```diff
@@ -0,0 +1,217 @@
+#include <platform.h>
+#include <backup.h>
+
+#if defined (__MINGW32__)
+
+int backup_main(int argc, char **argv)
+{
+    printf("backup not supported on Windows\n");
+    return 1;
+}
+
+#elif ! defined (LMDB)
+
+int backup_main(int argc, char **argv)
+{
+    printf("backup only implemented for LMDB.\n");
+    return 1;
+}
+
+#else
+
+#include <stdio.h>
+
+bool copy_file(const char *src, const char *dst)
+{
+    assert(src != NULL);
+    assert(dst != NULL);
+
+    printf("Copying: '%s' -> '%s'\n", src, dst);
+    FILE *in = fopen(src, "r");
+    if (in == NULL)
+    {
+        printf("Could not open '%s' (%s)\n", src, strerror(errno));
+        return false;
+    }
+    FILE *out = fopen(dst, "w");
+    if (out == NULL)
+    {
+        printf("Could not open '%s' (%s)\n", dst, strerror(errno));
+        fclose(in);
+        return false;
+    }
+    size_t read = 0;
```

Comment:
No warnings in static analysis that this is masking a function?

Reply:
Nope ;)